torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -69,32 +20,64 @@ defmodule Torchx.NxDoctestTest do
       Torchx.Backend.__unimplemented__()
       |> Enum.map(fn {fun, arity} -> {fun, arity - 1} end)
       |> Kernel.++(
```

Comment:
Two additional suggestions:

1. What if we break this down in two categories: a) can't be fixed because it is an inherent limitation, like round or b) it is temporarily broken or needs more investigation, like dot, slice_axis, slice, all_close?/2, etc 

2. Then inside each category we order alphabetically

Reply:
That's a good idea :)